component/home/home.css

```diff
@@ -0,0 +1,225 @@
+@import url("https://fonts.googleapis.com/css?family=Lato:400,700");
+
+:root{
+    --primaryColor: #1276a5;
+    --secondaryColor:#cf443f;
+    --mainWhite: #fff;
+    --mainBlack: #222;
+    --mainGrey: #ececec;
+    --mainSpacing: 0.1rem;
+    --main-bg:#043044;
+}
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  body {
+    font-family: "Lato", sans-serif;
+  }
+
+  /* navbar starts */
+
+  .docnav-bar {
+	background: var(--main-bg);
+    color: var(--mainWhite);
+    position: sticky;
+    top: 0;
+    height: 4.5rem;
+    width: 100%;
+	margin-left: 0 auto;
+    justify-content:space-evenly;
+    z-index: 1;
+  }
+  .flex-center{
+	display: flex;
+	place-items: center center;
+	flex-wrap: wrap;
+  }
+  .nav-icon{
+	  font-size: 1.4rem;
+  }
+
+  .title-firstword{
+    color: var(--primaryColor);
+  }
+  .title-secondword{
+    color: var(--secondaryColor);
+  }
+  .nav-title{
+	text-transform: uppercase;
+	font-size: 1.5rem;
+	padding:0 1rem;
+  }
+ .navbar h1:hover{
+    border:1px solid black;
+    padding:8px;
```

Comment:
Just a suggestion, despite the value being small, still using rem might be a good practice.

Reply:
okay i update it